src/main/java/com/empowerhack/family/memberservice/Entity/Member.java

```diff
@@ -0,0 +1,31 @@
+package com.empowerhack.family.memberservice.Entity;
+
+import lombok.Data;
+
+import javax.persistence.*;
+import javax.validation.constraints.*;
+import java.util.Set;
+
+@Data
+@Entity
+final public class Member {
```

Comment:
Good question. I would not say **never**, but ideally **no**. Initially I want to take a strict approach and if we need to relax later on, then we can. Similar to the `private / protected / public` debates :smile: